thisfinans/src/main/java/com/programeiros/thisfinans/model/entities/ENUM/UserType.java

```diff
@@ -0,0 +1,6 @@
+package com.programeiros.thisfinans.model.entities.ENUM;
+
+public enum TypeUser {
```

Comment:
faltou renomear o nome aqui no enum, foi alterado apenas no nome do arquivo

Reply:
o projeto não está compilando

Reply:
https://www.facebook.com/IntelliJIDEA/videos/you-can-rename-packages-classes-interfaces-enums-or-variable-names-with-ease-in-/888467681568491/